### PR TITLE
feat: add common UI components

### PR DESCRIPTION
**Pull request by huynguyen-hl**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [x] 📜 README.md
- [x] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: packages/components/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "build:watch": "tsc -b --watch",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "test": "jest --ci --env=jest-environment-jsdom"
+    "test": "jest --ci --env=jest-environment-jsdom --collect-coverage"
```
Comment (namhoang1604) [inline packages/components/package.json:13]: Please one more script for coverage

File: packages/components/src/common-components/README.md
Comment (namhoang1604) [inline packages/components/src/common-components/README.md:1]: I think we don't need readme for common components

File: packages/components/src/common-components/ToastMessage/ToastMessage.tsx
Comment (namhoang1604) [inline packages/components/src/common-components/ToastMessage/ToastMessage.tsx:1]: How to use ToastMessage as a service? We want the ToastMessage to be set up at App and use its service to trigger messages. That should have the guideline.

**Review by namhoang1604 — CHANGES_REQUESTED** [review]
Can you please correct the commit message?

File: packages/components/src/common-components/ToastMessage/ToastMessage.tsx
Comment (huynguyen-hl) [reply packages/components/src/common-components/ToastMessage/ToastMessage.tsx:1]: I have just updated the ToastMessage component to be more convenient when setting up and triggering messages.

**Comment by huynguyen-hl** [comment]
> Can you please correct the commit message?

Yes, I just updated that commit

**Review by namhoang1604 — APPROVED** [review]
LGTM

**Review by ldhyen99 — APPROVED** [review]

**Review by namhoang1604 — APPROVED** [review]

**Review by ldhyen99 — APPROVED** [review]

**Pull request merged**